jobs/JEVS_CAM_PLOTS

```diff
@@ -65,10 +65,13 @@ if [ $MODELNAME = href ] ; then
 else
    if [ $VERIF_CASE = severe ] ; then
       export VDATE=${VDATE:-$PDYm7}
+      export EVAL_PERIOD=${EVAL_PERIOD:-LAST31DAYS}
    elif [ $VERIF_CASE = radar ] ; then
       export VDATE=${VDATE:-$PDYm2}
+      export EVAL_PERIOD=${EVAL_PERIOD:-LAST31DAYS}
    else
       export VDATE=${VDATE:-$PDYm1}
+      export EVAL_PERIOD=${EVAL_PERIOD:-last31days}
```

Comment:
Should all of the default values be lower case instead of all caps?
As ins should lies 68 and 71 match 74 (LAST31DAYS -> last31days)?

Reply:
Yes let me change that!